scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -59,24 +76,33 @@ export function RelatedItemsDirective(authoringWorkspace: AuthoringWorkspaceServ
                         .map((key) => scope.item.associations[key]);
 
                     const currentCount = relatedItemsForCurrentField.length;
-                    const maxCount = get(scope, 'field.field_options.multiple_items.enabled') === true
-                        ? get(scope, 'field.field_options.multiple_items.max_items')
+                    const maxCount = scope.field?.field_options?.multiple_items?.enabled === true
+                        ? scope.field.field_options.multiple_items.max_items
                         : 1;
 
                     const type = getSuperdeskType(event, false);
-                    const item = angular.fromJson(event.originalEvent.dataTransfer.getData(type));
+                    const item: IArticle = angular.fromJson(event.originalEvent.dataTransfer.getData(type));
 
                     const itemAlreadyAddedAsRelated = relatedItemsForCurrentField.some(
                         (relatedItem) => relatedItem._id === item._id,
                     );
 
+                    const isWorkflowAllowed = relationsService.itemHasAllowedStatus(item, scope.field);
+
+                    if (!isWorkflowAllowed) {
+                        notify.error(
+                            gettext('The following status is not allowed in this field: ') + `"${item.state}"`,
```

Comment:
Include the status in the translation string so it can appear in different positions in translations:

`gettext('The following status is not allowed in this field: {{status}}', {status: item.state})`

It would also help to display which statuses are allowed.

Reply:
great